meshtastic/tests/test_mesh_interface.py

```
@@ -2021,6 +2397,558 @@ def test_wait_helpers_raise_expected_timeout_errors() -> None:
+        with iface._response_handlers_lock:
+            assert (
+                "receivedTelemetry",
+                mesh_interface_module.UNSCOPED_WAIT_REQUEST_ID,
+            ) not in iface._response_wait_acks
+            assert (
+                "receivedTelemetry",
+                mesh_interface_module.UNSCOPED_WAIT_REQUEST_ID,
+            ) not in iface._response_wait_errors
+
+        iface._mark_wait_acknowledged("receivedTelemetry")
+        assert getattr(iface._acknowledgment, "receivedTelemetry") is True
```

Ruff (0.15.5)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)